README.md

```diff
@@ -9,6 +9,8 @@ C++ wrapper around the aws-c-* libraries. Provides Cross-Platform Transport Prot
 * aws-c-mqtt: MQTT client.
 * aws-c-auth: Auth signers such as Aws-auth sigv4
 * aws-c-http: HTTP 1.1 client, and websockets (H2 coming soon)
+* aws-checksums: Cross-Platform HW accelerated CRC32c and CRC32 with fallback to efficient SW implementations.
```

Comment:
Wasn't this merged into aws-c-common?

Reply:
Do you mean aws-checksums depends on aws-c-common now?

Reply:
Looks like you have the current state of things right.